bin/.gitignore

```diff
@@ -0,0 +1,4 @@
+*
+!.gitignore
+!setup
+!test_tasks
```

Comment:
Setup and test_tasks are shell scripts and can be kept. Also, could you add this to the `.gitignore` in the root of the project so they are all in one place?

Reply:
I'll move the .gitignore, sure.  That's what this is doing, though: ignoring everything, and excluding those from the ignorance.

Reply:
Oh I see. I've never used this style before, but that makes sense 👍 Thanks for moving it